department-of-veterans-affairs/vets-website
# Pension 69961 mvp current spouse monthly support

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Health and employment information' chapter

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Household Information' chapter
4. Select 'Married' or 'Separated' on the 'Marital status' page
5. Continue and complete the 'Current spouse's name' page
7. Continue and complete the 'Current marriage information' page
8. Continue and verify the 'Financial support for your spouse' page is next
9. Verify the content and design of the page
9. Confirm that the input is required
10. Verify that all changes are saved to the redux store in form data and included in the save form api call

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69961)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Financial support for your spouse | Financial support for your spouse (Required) |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_household_marital-status_spouse-monthly-support](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/97499317-6173-4dc7-8c89-4b455ca96428)|![localhost_3001_pension_application_527EZ_household_marital-status_spouse-monthly-support (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ca175d04-9c82-4e83-b45b-1b4e746dd582)


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69961)

### Quality Assurance & Testing

### Error Handling


### Authentication
